### PR TITLE
Update to PSY 2.0

### PR DESCRIPTION
**Pull request by rodrigomha**
Ready for review. Tests are passing locally with the latest changes in PSY and PF

### CODE REVIEW

File: src/base/device_wrapper.jl [hunk 1]
```diff
@@ -442,23 +442,19 @@ function StaticLoadWrapper(bus::PSY.Bus, loads::Vector{PSY.ElectricLoad}, bus_ix
...
-            elseif PSY.get_available(ld) &&
-                   PSY.get_model(ld) == PSY.LoadModels.ConstantPower
-                P_power += PSY.get_active_power(ld)
-                Q_power += PSY.get_reactive_power(ld)
-            else
-                error(
-                    "Not supported load model in $(PSY.get_number(bus)) named $(PSY.get_name(ld))",
-                )
-            end
+            P_power += PSY.get_active_power(ld)
```
Comment (jd-lara) [inline src/base/device_wrapper.jl:445]: lets implement this function with dispatch

File: src/base/perturbations.jl [hunk 2]
```diff
@@ -593,7 +593,22 @@ mutable struct LoadChange <: Perturbation
...
         ref_value::Float64,
     )
-        if signal ∉ [:P_ref, :Q_ref]
+        # Currently I'm assumming P_ref and Q_ref are constant impedance to 
+        if signal ∈ [:P_ref, :Q_ref]
+            @warn(
+                "P_ref and Q_ref signals will be deprecated. It will be assumed as a change in constant impedance for StandardLoads and a change in constant power for PowerLoads. Allowed signals are :P_ref_impedance, :Q_ref_impedance, :P_ref_power, :Q_ref_power, :P_ref_current, :Q_ref_current"
+            )
+        end
+        if signal ∉ [
```
Comment (jd-lara) [inline src/base/perturbations.jl:602]: Define signals as a const

File: src/base/device_wrapper.jl (hunk 1 above)
Comment (rodrigomha) [reply src/base/device_wrapper.jl:452]: TODO: Need to discuss this properly

File: src/base/perturbations.jl (hunk 2 above)
Comment (rodrigomha) [reply src/base/perturbations.jl:602]: In src/definitions?

Comment (jd-lara) [reply src/base/perturbations.jl:602]: yes, create a constant with this

File: src/base/device_wrapper.jl (hunk 1 above)
Comment (jd-lara) [reply src/base/device_wrapper.jl:452]: ok, tbd

**Comment by jd-lara** [comment]
Performances test expected to fail

**Pull request merged**